ElectronicObserver/Window/Tools/ExpeditionCheck/ExpeditionCheckWindow.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Window
```

Comment:
There are a few problems with the tool version currently. Because Window is used instead of WindowBase, Jot isn't tracking its position when closing/opening. Jot is also set up to track ExpeditionCheckView which means both the tool and view version are using the same config, which probably isn't a good idea.

I'd suggest removing the tool version entirely, because it doesn't really provide any benefit, the view can be used in a floating state just like the tool version, with the only difference being the window style.